_shared/project/Makefile

```diff
@@ -18,7 +18,7 @@ $(call help,make db,initialize the DB and upgrade it to the latest migration)
 db: args?=upgrade head
 db: python
 	@tox -qe dev --run-command 'python bin/make_db'
-	@tox -qe dev  --run-command 'alembic -c conf/alembic.ini $(args)'
+	@tox -qe dev  --run-command 'alembic $(args)'
```

Comment:
There's no need for the `-c conf/alembic.ini` anymore: I've moved it into an envvar in `tox.ini` so you no longer have to type it out every time you type an `alembic` command